test/models/user_test.rb

```diff
@@ -64,6 +64,11 @@ class UserTest < ActiveSupport::TestCase
       assert_equal 1, @user.call_list_patients('DC').count
     end
 
+    it 'should not fail if a patient gets unattached from the call list' do
+      @user.call_list_entries.where(line: 'DC').first.update patient: nil
```

Comment:
Is this specific to the DC line, or should it echo this behavior for MD and VA? (Or is it the same behavior; do we only need to test DC?)

Reply:
oh, this is just particular to the way we bootstrap this test (it also includes a MD line), but I guess it shouldn't matter since we're using assert diff. Fixing, thanks for this catch!